src/components/data/LocalVariablesContent.js

```diff
@@ -179,7 +218,28 @@ const LocalVariablesContent = ({ workspace, workspace: { workspace: { googleProj
                     value: editType,
                     onChange: ({ value }) => setEditType(value),
                     options: ['string', 'number', 'boolean', 'string list', 'number list', 'boolean list']
-                  }),
+                  })
+                ]) : null
```

Comment:
Instead of a ternary with null, you could just use `&&`

Reply:
Done.